polytracker/include/taintdag/taint.hpp

```diff
@@ -46,7 +46,7 @@ const source_index_t source_index_mask = max_source_index;
 
 // Use the remaining bits for source file offset
 const size_t source_offset_bits = storage_bits - source_index_bits - 2;
-using source_offset_t = storage_t;
+using source_offset_t = int64_t;
```

Comment:
Why change? We don't touch stdin at this point, right? I don't think it is a problem, just curious. If it is the output position we wan't to change there is a definition for that here: https://github.com/trailofbits/polytracker/blob/6477-add-stdout-stderr-as-optional-taint-sinks-behind-a-cli-flag/polytracker/include/taintdag/taint_sink_log.hpp#L9

Reply:
The reason I changed it for both sources and sinks is that the functions that, introduce taints (`pread()`) and sink taints (`fwrite()`) both use signed integers to represent offset. Don't know why sources do, but they do :shrug:

Reply:
OK, makes sense. Just hoping I haven't made any assumptions about having unsigned types...